### PR TITLE
📖 Update example1 and user quickstarts to use new command

### PR DESCRIPTION
**Pull request by MikeSpreitzer**
## Summary
This PR updates the example1 and user quickstarts to show the use of the new `kubestellar-list-syncing-objects` command.

This also fixes the command shown in example1 to wait for the placement translator to do its job to _only_ wait on that much.

## Related issue(s)

Fixes #

### CODE REVIEW

**Comment by MikeSpreitzer** [comment]
/cc @clubanderson 
/cc @KPRoche 
/cc @dumb0002 
/cc @francostellari 
@ezrasilvera

File: docs/content/Coding Milestones/PoC2023q1/example1-subs/example1-stage-3.md [hunk 1]
```diff
@@ -62,6 +62,50 @@ while ! kubectl get ct -n specialstuff my-new-cron-object; do sleep 10; done
...
+You can check that the common workload's ReplicaSet objects got to
+their mailbox workspaces with the following command. It will list the
+two copies of that object, each with an annotation whose key is
+`kcp.io/cluster` and whose value is the kcp `logicalcluster.Name` of
+the mailbox workspace; those names appear in the "CLUSTER" column of
+the custom-columns listing near the end of [the section above about
+the mailbox controller](../#the-mailbox-controller).
+
+```shell
+kubestellar-list-syncing-objects --api-group apps --api-kind ReplicaSet
```
Comment (clubanderson) [inline docs/content/Coding Milestones/PoC2023q1/example1-subs/example1-stage-3.md:74]: does this take wildcards?

Comment (MikeSpreitzer) [reply docs/content/Coding Milestones/PoC2023q1/example1-subs/example1-stage-3.md:74]: No.

**Comment by MikeSpreitzer** [comment]
The force-push to dccd404de revises the quickstart edit to make it not testable, since the quickstart is testing v0.9.0.

**Review by clubanderson — COMMENTED** [review]
/lgtm
/approve

**Comment by MikeSpreitzer** [comment]
No, it wasn't.  The force-push to 370f5bc53 actually has the change.

**Review by clubanderson — COMMENTED** [review]
/lgtm

**Pull request merged**